compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -281,7 +279,7 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
    // 32-bit platforms must check the processor and OS.
    // 64-bit platforms unconditionally support prefetching.
    //
-   if (_targetProcessorInfo.supportsSSE() && jitTestOSForSSESupport())
+   if (_targetProcessorInfo.supportsSSE() && TR::Compiler->target.cpu.testOSForSSESupport(comp))
```

Comment:
The only implementation of testOSForSSESupport I see in OMR unconditionally returns false.  Is that what it is supposed to do?

Reply:
Yes, now `testOSForSSESupport()` in OMR always returns false as it does not support SSE on X86-32 yet. Each downstream project can optionally override it to return true if the project properly support SSE. However, I believe there is an on-going effort to bring full SSE support to OMR on X86-32 and `testOSForSSESupport()` should give the correct answer after that.

Reply:
cool thanks for the info.